CHANGELOG.md

```diff
@@ -24,8 +24,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Changed path definitions for each area in all renditions of the map, resulting in a ~30% decrease in the overall SVG size.
 - Changed the name of fields in the area types as follows:
   - `en_name` -> `asciiName`
-  - `display_name` -> `displayName`
-  - `en_term` -> `enTerm`
```

Comment:
We don't need to change the name here since we are informing people of the change (:

Reply:
Ctrl & Shift & F did the magic trick 😅

Reply:
Caught in the cross-fire!